lib/repetitive-task.js

```diff
@@ -52,20 +53,26 @@ class RepetitiveTask {
   }
 
   __process () {
+    let processStartTime
     this.isProcessing = this._getTask()
       .then((task) => {
         if (!task) {
           this.logger.info('No pending tasks')
           return
         }
-
+        this.currentTask = task
         this.logger.info(this._getLogData(task), 'processing task')
+        processStartTime = process.hrtime()
         return this._process(task)
       })
       .catch((error) => {
         this.logger.error({ stack: error.stack }, 'Error processing task')
       })
       .then(() => {
+        const processElapsedTime = { elapsed_time: hrtimeToMs(process.hrtime(processStartTime)) }
+        const logInfo = Object.assign(this._getLogData(this.currentTask), processElapsedTime)
```

Comment:
Figured it would be a good idea to relate this output back to the processing task output via the task. For example if _getTask returned some sort of ID if would be in both outputs